sunscreen_compiler_common/src/context.rs

```diff
@@ -134,19 +197,51 @@ impl Render for EdgeInfo {
 #[derive(Clone, Deserialize, Serialize)]
 /**
  * The result of a frontend compiler.
+ *
```

Comment:
Don't put notes for yourself in Doc comments, as we don't want to accidentally ship these!